Firestore/Example/Tests/SpecTests/FSTSyncEngineTestDriver.h

```diff
@@ -31,6 +32,7 @@
 #include "Firestore/core/src/firebase/firestore/model/mutation.h"
 #include "Firestore/core/src/firebase/firestore/model/snapshot_version.h"
 #include "Firestore/core/src/firebase/firestore/model/types.h"
+#include "Firestore/core/src/firebase/firestore/nanopb/nanopb_util.h"
```

Comment:
Include nanopb/byte_string.h to get a declaration of `ByteString`.

Reply:
Done.